documentation/ansible-playbook-fully-disconnected.adoc

```diff
@@ -0,0 +1,34 @@
+[id="ansible-playbook-fully-disconnected"]
+
+= Fully Disconnected Deployment of IPI on BM using the Ansible Playbook
+Deployment Integration Team
+:data-uri:
+:email: https://github.com/openshift-kni/baremetal-deploy
+:experimental:
+:toc-title:
+:imagesdir: images
```

Comment:
images I think that means documentations/images so it could be crowded with other manuals we can tune that to be the ansible-XXX/images folder?